frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "map-of-pi",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
This is a good point!  What are your thoughts about automating the increment of the version number?

Reply:
Absolutely, would be useful..